### DISCUSSION TITLE
Cure Model Interpretation

### DISCUSSION THREAD

**Discussion by helloannietran** [CamDavidsonPilon/lifelines#1264] [Q&A] [answered]
Hi @CamDavidsonPilon! I'm experimenting with using survival regression with a cure component to model the lifetimes of customers that we identify as at risk of churning, and I need help with interpreting the summary of the cure regression model 😅. The dataset contains all accounts that we identify as "at-risk" and some of them return and some don't (churn) so the asymptote in the survival curve, in this case, is the churn rate. 

The custom function pretty much looks like the one in the example in the doc:

```
class CureModel(ParametricRegressionFitter):
    _scipy_fit_method = "SLSQP"
    _scipy_fit_options = {"ftol": 1e-10, "maxiter": 200}

    _fitted_parameter_names = ["lambda_", "beta_", "rho_"]

    def _cumulative_hazard(self, params, T, Xs):
        c = expit(np.dot(Xs["beta_"], params["beta_"]))

        lambda_ = np.exp(np.dot(Xs["lambda_"], params["lambda_"]))
        rho_ = np.exp(np.dot(Xs["rho_"], params["rho_"]))
        sf = np.exp(-(T / lambda_) ** rho_)

        return -np.log(c + (1-c) * sf)
```

I think I know how to interpret beta in the summary since it represents the churn rate, but I also have features added to tune lambda, and I'm not sure how to correctly interpret the effect of the coefficients for lambda. I've seen cases where the coefficient is positive for the same feature for beta but negative for lambda. Any guidance + resources to help there would be super helpful!

PS: Thanks so much for writing this package. It's cool stuff!

**Comment by CamDavidsonPilon** [comment, marked as answer]
hi @helloannietran - I can try to help here. I don't have any canonical resource I can share, but I'll give you a better feeling for the model. 

1. When the coefficient has opposite signs in `lambda` vs `beta`, consider also the std. error (in the `se(coef)` column). If the `se(coef)` is large relative to the `coef`, then the difference of signs could be just noise.
2. If you are truly interested in comparing coefficients, you need to consider the entire causal DAG and not fall for the [Table 2 fallacy](http://dagitty.net/learn/graphs/table2-fallacy.html). That being said, all interpretation is meaningless if you don't have a specified DAG.  
2. Let's think carefully about what a positive coefficient in `beta` means: if a coef is positive, then an increase in the corresponding covariate puts the subject more often into the "cured" bucket, i.e. will never die. Now `lambda`: Your `sf` model is the Weibull model, which is an [AFT model](https://lifelines.readthedocs.io/en/latest/Survival%20Regression.html#accelerated-failure-time-models). If a coef is positive, then an increase in the corresponding covariate will "dilate" time (since we divide `T` by `exp(lambda_)`). This means that the covariate is protective.

**Reply by helloannietran** [reply]
Thanks, @CamDavidsonPilon! This helps clear things up a lot. Was also able to visualize point 3 with plot_partial_effects_on_outcome().
